README.md

```diff
@@ -41,3 +56,13 @@ This project provides the core ApostropheCMS widgets, plus seven additional widg
     - link-widget: adds links that can be styled as text or a highly customizable button
 
 ### Pieces
+This project creates two pieces. The first is an `article` piece for the creation of content pieces like blog posts or news articles. The second is an `author` piece that is used in relationship with the article pieces.
+
+### Pages
+This project creates core `default`, `@apostrophecms/home-page` and a 404 page. It also creates two pages for displaying the article pieces.
+
+The home-page has three potential layouts selected from the utility menus on the right-side of the page manager. The 'Minimal' layout inherits the header and footer components that is added to all the project pages. It also has a single area that can take any of the project widgets. The 'Foundation' layout adds a hero section at the top of the page, while the 'Showcase' adds a slideshow.
+
+The default page has a layout that is identical to the 'Minimal' home-page layout.
+
+Piece-type pages in ApostropheCMS are used to either display multiple pieces (`index.html`) or individual pieces (`show.html`). This project has both types of pages, each with three layouts. Depending on the index layout, there are three or four additional areas for adding widgets with content before and after the piece content.
```

Comment:
I would clarify that there won't be an `index.html` and `show.html` and show what there is instead.